src/inject.js

```diff
@@ -203,9 +203,13 @@ export const reset = () => {
     styleTag = null;
 };
 
+export const resetInjected = (key /* : string */) => {
```

Comment:
I expected this function to reset all injected styles, not one specific style. Perhaps a better name would be `resetInjectedStyle` or something that tells consumers it only resets a single one?

Reply:
Done!